utils/constants.py

```diff
@@ -34,10 +35,14 @@
 
 
 class DataType(ValueEnum):
-    """Swagger data types that we expect to deal with. Uses camelcase in values to match."""
+    """
+    Swagger data types that we expect to deal with. The latest pattern uses camelCase, while the older pattern uses snake_case.
+    """
 
     FIRST_NAME = "firstName"
+    FIRST_NAME_VARIANT = "first_name"  # matches older swagger format
     LAST_NAME = "lastName"
+    LAST_NAME_VARIANT = "last_name"  # matches older swagger format
```

Comment:
I know James added middle name and suffix to the fake data spreadsheet probably since this was last touched if we want to take advantage of those down the line.